MetaMorpheus/TaskLayer/CalibrationTask/CalibrationTask.cs

```diff
@@ -83,10 +83,22 @@ protected override MyTaskResults RunSpecific(string OutputFolder, List<DbForTask
 
                 // get filename stuff
                 string originalUncalibratedFilePath = currentRawFileList[spectraFileIndex];
+                var fileExtension = Path.GetExtension(originalUncalibratedFilePath);
                 string originalUncalibratedFilenameWithoutExtension = Path.GetFileNameWithoutExtension(originalUncalibratedFilePath);
+                bool calibrated = false;
+                if (fileExtension.Equals(".mgf", StringComparison.OrdinalIgnoreCase) || fileExtension.Equals(".d", StringComparison.OrdinalIgnoreCase))
```

Comment:
Did we calibrate mgf in the past? What would happen?

Reply:
Previously, there was a warning that popped up that .mgfs can't be calibrated, but there were no checks to prevent it.